src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTestFiller.json

```diff
@@ -0,0 +1,86 @@
+{
+    "doubleSelfdestructTest" : {
+        "info" : {
+            "comment00" : "The first test case required here https://github.com/ethereum/tests/issues/431#issue-306081539 .",
```

Comment:
such comment is not being copied to the final test. try using .yml for this description

Reply:
That's a good idea.

Reply:
Should the filler named `doubleSelfdestructTestFiller.yaml`?

Reply:
*.yml